test/suite/test_rollback_to_stable10.py

```diff
@@ -333,6 +339,7 @@ def prepare_range_updates(session, cursor, ds, value, nrows, prepare_config):
 
         stat_cursor = self.session.open_cursor('statistics:', None, None)
         calls = stat_cursor[stat.conn.txn_rts][2]
+        hs_file_size_after_crash = stat_cursor[stat.conn.cache_hs_ondisk][2]
```

Comment:
I think we need to grab this value immediately after the restart even though it means opening the statistics cursor twice (unless all those values can be retrieved immediately after restart). The `check` calls will open cursors and look at data  and could skew the results.